### ISSUE TITLE
Cannot find module 'particle-cli-library-manager'

### ISSUE THREAD

**Issue by towynlin** [spark/particle-cli#274]
As of commit c91f44f42e2067ed6c2ed3526473a3e344bfa9b3 from July 25th on the `develop` branch, using node v6.3.1 and npm 3.10.3 on a macOS Sierra Version 10.12 Beta (16A294a), I get the following error when running `npm test` after a successful clean `npm install`:

```
% npm test

> particle-cli@1.16.0 test /Users/zac/code/particle/particle-cli
> mocha test/ test/oldcmd/ test/app test/cli test/cmd test/integration --compilers js:babel-register

module.js:442
    throw err;
    ^

Error: Cannot find module 'particle-cli-library-manager'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/zac/code/particle/particle-cli/src/cmd/library_migrate.js:3:1)
    at Module._compile (module.js:541:32)
    at loader (/Users/zac/code/particle/particle-cli/node_modules/babel-register/lib/node.js:146:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/zac/code/particle/particle-cli/node_modules/babel-register/lib/node.js:156:7)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/zac/code/particle/particle-cli/src/cli/library_migrate.js:1:1)
    at Module._compile (module.js:541:32)
    at loader (/Users/zac/code/particle/particle-cli/node_modules/babel-register/lib/node.js:146:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/zac/code/particle/particle-cli/node_modules/babel-register/lib/node.js:156:7)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/zac/code/particle/particle-cli/test/cli/library.spec.js:21:1)
    at Module._compile (module.js:541:32)
    at loader (/Users/zac/code/particle/particle-cli/node_modules/babel-register/lib/node.js:146:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/zac/code/particle/particle-cli/node_modules/babel-register/lib/node.js:156:7)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at /Users/zac/code/particle/particle-cli/node_modules/mocha/lib/mocha.js:220:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/zac/code/particle/particle-cli/node_modules/mocha/lib/mocha.js:217:14)
    at Mocha.run (/Users/zac/code/particle/particle-cli/node_modules/mocha/lib/mocha.js:469:10)
    at Object.<anonymous> (/Users/zac/code/particle/particle-cli/node_modules/mocha/bin/_mocha:404:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
npm ERR! Test failed.  See above for more details.
```

**Comment by towynlin** [comment]
Actually this error happens even when running `particle` or `particle list`.

**Comment by towynlin** [comment]
Sorry — I hadn't done `git pull` — didn't realize I already had checkout out `develop` in the past — thought it was a new branch on my machine. Fixed in latest `develop` as of commit 77701f355b08e45c09a0b61ebc5cbea087f3976f from August 29.
